server/src/main/java/io/crate/analyze/AnalyzedCopyFrom.java

```diff
@@ -70,6 +71,14 @@ public Symbol uri() {
         return uri;
     }
 
+    public boolean waitForCompletion() {
+        return waitForCompletion;
+    }
+
+    public void setWaitForCompletion(boolean waitForCompletion) {
```

Comment:
A analysed statement should be immutable

Reply:
@mfussenegger Ok the set method will be removed.
I ended up having this set method because i was trying to find an optimal way to get the wait_for_completion setting once, after bind method evaluates symbols. Since bind method is static what do you think is the best practice would be to have the wait_for_completion value inside executeOrFail method of copy plans?